go-controller/pkg/node/secondary_node_network_controller.go

```diff
@@ -57,3 +66,7 @@ func (nc *SecondaryNodeNetworkController) Stop() {
 func (nc *SecondaryNodeNetworkController) Cleanup() error {
 	return nil
 }
+
+func (nc *SecondaryNodeNetworkController) ensureNetworkID() error {
+	return annotation.EnsureNetworkID(nc.watchFactory, nc.NetInfo, &nc.networkID)
```

Comment:
can you add UTs? I am rebased on top of this but always nc.networkID is giving me 0 even after ensureNetworkID() is called